### PR TITLE
Oded/tfl 1317

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: src/components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <span translate=".UPGRADE_TEXT_SUBJECT_{{$ctrl.summaryExerciseData.exercise.subjectId}}"></span>
+    <md-button
+        ng-click="$ctrl.showPurchaseDialog()"
+        class="znk outline"
+        translate=".UPGRADE_BTN">
+    </md-button>
+</div>
+
+<!--<div class="overlay-container pending-evaluation" ng-switch-when="pendingEvaluation">-->
+    <!--<div class="pending-evaluation-raccoon"></div>-->
```
Comment (assafshp) [inline src/components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html]: these comments are relevant for future evaluator ?

File: src/components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div class="results-wrapper znk-scrollbar base">
+    <div ng-repeat="questionResult in $ctrl.summaryExerciseData.exerciseResult.questionResults">
+        <evaluate-question-result
+            question-result="{
+                      index: $index + 1,
+                      userAnswer: questionResult.userAnswer
+                   }" exercise-type="$ctrl.summaryExerciseData.exercise.subjectId">
```
Comment (assafshp) [inline src/components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html]: are you sure that exercise-type is the subjectId

File: src/components/znkToeflSummary/directives/znkToeflSummaryEvaluatorDrv.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    angular.module('znk.infra-toefl.znkToeflSummary').component('znkToeflSummaryEvaluator', {
+        templateUrl: 'components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html',
+        bindings: {
+            summaryExerciseData: '<',
+            clickReview: '&'
+        },
+        controller: function(purchaseService) {
+            'ngInject';
+            this.showPurchaseDialog = function () {
+                purchaseService.showPurchaseDialog();
```
Comment (assafshp) [inline src/components/znkToeflSummary/directives/znkToeflSummaryEvaluatorDrv.js]: just make sure it uses the new functionality in the purchase that Oded did

File: src/components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html (hunk 2 above)
Comment (odeds) [inline src/components/znkToeflSummary/templates/znkToeflSummaryEvaluator.html]: ye it's ok

File: src/components/znkToeflSummary/directives/znkToeflSummaryEvaluatorDrv.js (hunk 3 above)
Comment (odeds) [inline src/components/znkToeflSummary/directives/znkToeflSummaryEvaluatorDrv.js]: ye it's exist

**Pull request merged**